### DISCUSSION TITLE
alembic won't run imgration fastapi_users_db_sqlalchemy is not defined

### DISCUSSION THREAD

**Discussion by SnoozeFreddo** [fastapi-users/fastapi-users#1373] [Q&A]
/alembic/versions/7a47ff544945_initial_migration.py", line 47, in upgrade
    sa.Column('id', fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False),
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'fastapi_users_db_sqlalchemy' is not defined

The migration file appears to be missing an import statement. Should I add it manually, or have I made an error elsewhere?

That's my User model:

```
from fastapi_users.db import SQLAlchemyBaseUserTableUUID
from myproject.bases import Base

class User(SQLAlchemyBaseUserTableUUID, Base):
    ...
```

**Comment by cerealcable** [comment]
I've been adding this import myself. Alembic is generating these migrations not fastapi-users, so not sure if anything can be done from the perspective of fastapi-users.

**Comment by hgalytoby** [comment]
This is not a bug in `fastapi-users`.

If you use Field as defined by the `sqlalchemy` other packages, you have to modify `script.py.mako`.

Add the required import.

## alembic/script.py.mako
```
"""${message}

Revision ID: ${up_revision}
Revises: ${down_revision | comma,n}
Create Date: ${create_date}

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa
import fastapi_users_db_sqlalchemy   # Add this import

${imports if imports else ""}

# revision identifiers, used by Alembic.
revision: str = ${repr(up_revision)}
down_revision: Union[str, None] = ${repr(down_revision)}
branch_labels: Union[str, Sequence[str], None] = ${repr(branch_labels)}
depends_on: Union[str, Sequence[str], None] = ${repr(depends_on)}


def upgrade() -> None:
    ${upgrades if upgrades else "pass"}


def downgrade() -> None:
    ${downgrades if downgrades else "pass"}

```
